cmd/kube-apiserver/app/options/globalflags_nolegacyproviders.go

```diff
@@ -0,0 +1,27 @@
+// +build providerless
```

Comment:
OTOH "noproviderless" is going to be an unfortunate file name :/

Reply:
providers / providerless?

I can rework the PR, may take a bit :grimacing:

Reply:
Yeah, sorry, it just kind of jumped out at me, I don't have any good suggestions. "providerlessasinnoproviders" and "providerlessasinserverless"